.github/workflows/wash.yml

```diff
@@ -26,17 +26,17 @@ jobs:
     strategy:
       fail-fast: false # Ensure we can run the full suite even if one OS fails
       matrix:
-        os: [ubuntu-22.04, windows-latest-8-cores, macos-12]
+        os: [ubuntu-22.04, windows-latest, macos-12]
```

Comment:
Consider bumping `macos-12` here to [`macos-13` or `macos-14`](https://github.com/actions/runner-images/tree/main?tab=readme-ov-file#available-images)

Reply:
Will bump to 13 here since 14 changes to the arm runners, and I don't want to accidentally do what wadm did to break x86 builds